security/DOXM-OwnerCreds.txt

```diff
@@ -44,8 +44,10 @@ OT->ND: POST /oic/sec/pstat[{..., "cm:":"bx0011,1100",...}]
 ND->OT: RSP 2.04
 OT->ND: POST /oic/sec/pstat[{..., "isop:":"TRUE",...}]
 ND->OT: RSP 2.04
-OT->ND: Close Secure Session
-OT->ND: Open Secure Session with new Owner Credentials
+
+note over OT, ND
+Provision the new device with additional credentials for device management and device-to-device communications. 
+end note
```

Comment:
Well, there was a separate one for 1339 that changed this same section in the JW sequence. The one for cert OTM was swept up in this one. A little weird - but end result is fine...